app/Http/Client/Alchemy/AlchemyPendingRequest.php

```diff
@@ -685,13 +707,13 @@ private function getNftV2ApiUrl(): string
         return 'https://'.self::$apiUrlPlaceholder.'.g.alchemy.com/nft/v2/';
     }
 
-    private function filterNft(mixed $nft): bool
+    private function filterNft(mixed $nft, bool $filterError = true): bool
```

Comment:
what is `filterError` for? I see it used once here https://github.com/ArdentHQ/dashbrd/pull/230/files#diff-957272cbfdd990765850dc7924f1ac272f6ea71fb3403acbe633fe548369ebf6R213 but that doesn't make it clear to me when to pass that value 🤔